src/explore-education-statistics-admin/src/pages/EditReleasePage.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import Page from '@admin/components/Page';
+import { Release } from '@admin/services/publicationService';
+import DummyPublicationsData from '@admin/pages/DummyPublicationsData';
+import EditReleaseSetupSummary from '@admin/components/EditReleaseSetupSummary';
+import NavigableSections, {
+  Section,
+} from '@admin/components/NavigableSections';
+import { RouteComponentProps } from 'react-router';
+
+export enum ReleaseSection {
+  ReleaseSetup,
+  AddEditData,
+  BuildTables,
+  ViewEditTables,
+  AddEditContent,
+  SetPublishStatus,
+}
+
+interface Props extends RouteComponentProps {
+  releaseId: string;
+}
+
+const navigationHeadings: (
+  releaseId: string,
+) => Section<ReleaseSection>[] = releaseId => {
+  const urlPrefix = `/edit-release/${releaseId}`;
+  return [
+    {
+      section: ReleaseSection.ReleaseSetup,
+      label: 'Release setup',
+      linkTo: `${urlPrefix}/setup`,
+    },
+    {
+      section: ReleaseSection.AddEditData,
+      label: 'Add / edit data',
+      linkTo: `${urlPrefix}/data`,
+    },
+    {
+      section: ReleaseSection.BuildTables,
+      label: 'Build tables',
+      linkTo: `${urlPrefix}/build-tables`,
+    },
+    {
+      section: ReleaseSection.ViewEditTables,
+      label: 'View / edit tables',
+      linkTo: `${urlPrefix}/tables`,
+    },
+    {
+      section: ReleaseSection.AddEditContent,
+      label: 'Add / edit content',
+      linkTo: `${urlPrefix}/content`,
+    },
+    {
+      section: ReleaseSection.SetPublishStatus,
+      label: 'Set publish status',
+      linkTo: `${urlPrefix}/publish-status`,
+    },
+  ];
+};
+
+const breadcrumbs = [
```

Comment:
Would inline this

Reply:
It looked really scrappy inline - felt it read a lot better when pulling it out tbh

Reply:
Yeh I mean it's not a big deal, but generally the breadcrumbs just get inlined in other components. Additionally, you often even need the breadcrumbs to at least be inside of the component function to get various variables in scope.

Reply:
Done